scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -476,7 +476,21 @@ package com.twitter.scalding {
     }
   }
 
-  /** In the typed API every reduce operation is handled by this Buffer */
+  private[scalding] object SkewMonitorCounters {
+    // Strangely, if group name and key name are different, then the counter would be zero
```

Comment:
this is bug by itself. Can you confirm this is still true and not due to another issue? We don't want to make a bunch of groups.